go.mod

```diff
@@ -1,19 +1,40 @@
 module github.com/scaleway/terraform-provider-scaleway
 
 require (
-	github.com/aws/aws-sdk-go v1.39.3
+	cloud.google.com/go/storage v1.16.0 // indirect
+	github.com/agext/levenshtein v1.2.3 // indirect
+	github.com/aws/aws-sdk-go v1.38.69
```

Comment:
Why was the aws-sdk-go downgraded?

Reply:
resolved, the dep was updated (and weirdly downgraded) when adding a UUID dep, but I'm not using it anymore